docs/standard/serialization/system-text-json-converters-how-to.md

```diff
@@ -267,132 +104,55 @@ If you do provide a message (for example, `throw new JsonException("Error occurr
 
 ## Registration sample - Converters collection 
 
-Here's an example that makes the `ExampleDateTimeOffsetConverter` the default for properties of type `DateTimeOffset`:
+Here's an example that makes the `DateTimeOffsetConverter` the default for properties of type `DateTimeOffset`:
```

Comment:
It would be good to include link to the APIs, such as:

```suggestion
Here's an example that makes the <xref:System.ComponentModel.DateTimeOffsetConverter> the default for properties of type <xref:System.DateTimeOffset>:
```